### ISSUE TITLE
Transfer from Ropsten Testnet to Rinkeby Testnet

### ISSUE THREAD

**Issue by AlexJupiter** [DemocracyEarth/contracts#84]
The Ropsten testnet is failing like tired little lamb. 

After some research, it seems like the Rinkeby testnet (started by the Ethereum team) is the best alternative as it has the ideal blocktime of 15seconds. The Kovan testnet, the only other alternative, has a blocktime of 4 seconds, which is incredibly unrealistic.

**Comment by aecc** [comment]
@AlexJupiter yes, I agreed with my thumbs up some time ago. I think it makes sense. Multiple ppl have asked me why don't we use Rinkeby. Keep in mind we need to modify the README after this is done.
